README.md

```diff
@@ -57,6 +57,9 @@ kontent environment add --name DEV --api-key <Api key> --project-id <Project ID>
 npm run migrate 01_sample_init_createBlogType
 ```
 
+Kontent-CLI supports only running javascript migration files so in case you want to write in TypesSript, CoffeScript or in any other language you have to transpile your code before running.

```

Comment:
Why Kontent-CLI? Shouldn't it be Kontent CLI? You write javascript and then follow with CoffeeScript. Be consistent in capitalization and stick to WhateverScript format. Also, what's "TypesSript"? Use spellchecker, it saves lives.